packages/datagateway-dataview/src/views/detailsPanels/isis/datafileDetailsPanel.component.tsx

```diff
@@ -25,14 +25,14 @@ const useStyles = makeStyles((theme: Theme) =>
 
 interface DatafileDetailsPanelProps {
   rowData: Entity;
-  detailsPanelResize: () => void;
   fetchDetails: (datafileId: number) => Promise<void>;
+  detailsPanelResize: () => void;
```

Comment:
In several other places (e.g. `datasetDetailsPanel`) `detailsPanelResize`was made optional. It seems strange that this was altered but not made optional like elsewhere. There may be a good reason for this, but if not then maybe change this to be in line with the other detail panels?

Reply:
Hi @patrick-austin , the `detailsPanelResize` has not been made optional in datefile tables since there was the need to show information for each datafile in the table and a details panel would be mandatory in that case. It was previously mandatory for all tables but some details panel have been made optional so it is easier to create the card views as details panels aren't always required.

Reply:
That makes sense, thanks for the explanation.